### DISCUSSION TITLE
Add a "Save As" button

### DISCUSSION THREAD

**Discussion by ghost** [pyload/pyload#4094] [Suggestions]
Hello,
Is it possible to add a "Save As" button when adding a download link to download certain files to a specific folder?
Sorry to my bad English..

**Comment by GammaC0de** [comment]
pyLoad uses a client-server model where the download server can be on a remote site and the client right in front of you.
So because the server is on a remote site, you cannot choose "Save As".
What you can do is when the download finishes, you can go to the "downloads" tab an click on the downloaded file and save the to your disk.
